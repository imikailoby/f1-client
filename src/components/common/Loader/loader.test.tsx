/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import { Loader } from './';

describe('<Loader />', () => {
  it('renders without crashing', () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});
