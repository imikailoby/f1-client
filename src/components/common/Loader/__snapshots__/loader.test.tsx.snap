// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders without crashing 1`] = `
<div>
  <div
    class="loader"
  >
    <img
      alt="loader"
      src="svg>
"
    />
  </div>
</div>
`;
